base/stl_helpers.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "std/algorithm.hpp"
+#include "std/auto_ptr.hpp"
```

Comment:
> Sorry, I didn't get your answer. Could you please paraphrase it.

It's because github changed my line:) I wrote: "We include < memory > in auto_ptr.hpp. It's necessary for allocator." And memory header contains declaration of allocator. So I wrote  include < memory > to use allocator next.